support-frontend/assets/pages/contributions-landing/contributionsLanding.scss

```diff
@@ -42,6 +42,20 @@ body {
   }
 }
 
+.gu-content--contribution-thankyou.gu-content--aus-moment {
+  background: linear-gradient(
```

Comment:
is this the same as the background styling of the other page? Can they share the css?

Reply:
It is, they can, and they now do! Thanks @tomrf1 🙂

Reply:
great!